src/blocks/Contact.astro

```diff
@@ -41,32 +42,35 @@ const {
     <Row>
       {buttons?.map((button: any) => <Button {...button} />)}
     </Row>
+
     <Column>
       <Channel
```

Comment:
Do the following with the channel component:

- add a type prop: 'tel' | 'email' | 'address'
- if one of those type is set, you can automatically determine the heading,icon and href based on the text being passed. If not, it is just an object.